tests/test_integration.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+
+from datasets import load_dataset
+from sklearn.datasets import fetch_20newsgroups
+from sklearn.feature_extraction.text import CountVectorizer
+
+from topic_benchmark.cli import run_cli
+from topic_benchmark.datasets.dataset import Dataset
+from topic_benchmark.registries import dataset_registry
+
+
+@dataset_registry.register("dummy_ikea")
+def load_ikea():
+    ds = load_dataset("nbadrinath/ikea_dataset_5.0", split="test[:64]")
+    images = ds["image"]
+    texts = ds["desc"]
+    nans = []
+    for image, text in zip(images, texts):
+        if (image is None) or (text is None):
+            nans.append(True)
+        else:
+            nans.append(False)
+    images = [image for image, is_nan in zip(images, nans) if not is_nan]
+    texts = [text for text, is_nan in zip(texts, nans) if not is_nan]
+    return Dataset(list(texts), images=list(images))
+
+
+@dataset_registry.register("dummy_20NG")
+def load_newsgroups_raw() -> list[str]:
+    ds = fetch_20newsgroups(subset="all")
+    corpus = list(ds.data)[:32]
+    return Dataset(corpus)
+
+
+OUT_FOLDER = Path(__file__).parent.joinpath("__test_results")
```

Comment:
```suggestion
OUT_FOLDER = Path(__file__).parent / "__test_results"
```
Not sure if you know that this is an option?

Reply:
Yes I have known about it for a while. I just don't like it for some reason. Like in my head it doesn't make sense that the same operator is used for division as for adding paths together, like shouldn't this be a `+` instead?? 

Anyway... I prefer to spell it out, it doesn't hurt anyone